app/api/business/evidence_business.py

```diff
@@ -45,3 +46,29 @@ def delete_draft_evidence(evidence_id, actioned_by):
         db_object=evidence
     )
     return True
+
+
+def case_studies_by_supplier_code(supplier_code, domain_id):
+    case_studies = {}
+    case_studies['data'] = evidence_service.get_case_studies_by_supplier_code(supplier_code, domain_id)
+    # simplify case_studies data structure
+    get_case_study_data = case_studies.get('data')
+    retrieve_list = get_case_study_data[0]
+    case_studies = retrieve_list[0]
+    return case_studies
+
+
+def get_domain_and_evidence_data(evidence_id):
+    evidence = evidence_service.get_evidence_by_id(evidence_id)
+    data = {}
+    data = evidence.serialize()
+    data['domain_name'] = evidence.domain.name
+
+    domain_criteria = domain_criteria_service.get_criteria_by_domain_id(evidence.domain.id)
```

Comment:
Did you think about using one query (service call) to return the relevant data?  We probably don't need to return all the criteria do we (only what has been selected by the seller)?